registry-library/library/library.go

```diff
@@ -349,7 +350,7 @@ func PullStackFromRegistry(registry string, stack string, destDir string, option
 }
 
 // decompress extracts the archive file
-func decompress(targetDir string, tarFile string) error {
+func decompress(targetDir string, tarFile string, excludeFiles []string) error {
```

Comment:
Could you declare the exclude list in this block? https://github.com/devfile/registry-support/blob/c45fcd671516034dd299cfce2e7db718fefa3602/registry-library/library/library.go#L54